<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Progress</class>
 <widget class="QWidget" name="Progress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>62</y>
     <width>300</width>
     <height>6</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QProgressBar{
    color : solid gray;
    border: 1px solid gray;
    border-radius: 2px;
    background: transparent;
    padding: 0px;
    text-align : center ;
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>35</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>STATUS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 常规</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_ok{

background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid #555;
}


QPushButton#pushButton_ok:hover{
background-color: rgb(0, 255, 127);
	color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid #555;
}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 233, 239, 255), stop:1 rgba(157, 226, 255, 255));

border-radius:20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>35</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <property name="text">
    <string>WAIT</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>progressBar</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_ok</zorder>
  <zorder>label_status</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
